docs/integrations/sources/s3.md

```diff
@@ -3,46 +3,66 @@
-<!-- /env:cloud -->
+:::caution
+Your `Secret Access Key` will only be visible once upon creation. Be sure to copy and store it securely for future use.
+:::
 
-<!-- env:oss -->
+For more information on managing your access keys, please refer to the 
+[official AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
 
-**For Airbyte Open Source:**
+### Step 2: Set up the Amazon S3 connector in Airbyte
 
-1. Create a new S3 source with a suitable name. Since each S3 source maps to just a single table, it may be worth including that in the name.
-2. Set `dataset` appropriately. This will be the name of the table in the destination.
-3. If your bucket contains _only_ files containing data for this table, use `**` as path_pattern. See the [Path Patterns section](s3.md#path-patterns) for more specific pattern matching.
-4. Leave schema as `{}` to automatically infer it from the file\(s\). For details on providing a schema, see the [User Schema section](s3.md#user-schema).
-5. Fill in the fields within the provider box appropriately. If your bucket is not public, add [credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) with sufficient permissions under `aws_access_key_id` and `aws_secret_access_key`.
-6. Choose the format corresponding to the format of your files and fill in fields as required. If unsure about values, try out the defaults and come back if needed. Find details on these settings [here](s3.md#file-format-settings).
-<!-- /env:oss -->
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account, or navigate to your Airbyte Open Source dashboard.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New source**.
+3. Find and select **S3** from the list of available sources.
+4. Enter the **Output Stream Name**. This will be the name of the table in the destination (can contain letters, numbers and underscores).
+5. Enter the **Pattern of files to replicate**. This is a regular expression that allows Airbyte to pattern match the specific files to replicate. If you are replicating all the files within your bucket, use `**` as the pattern. For more precise pattern matching options, refer to the [Path Patterns section](#path-patterns) below.
+6. Enter the name of the **Bucket** containing your files to replicate.
+7. Toggle the **Optional fields** under the **Bucket** field to expand additional configuration options. **If you are syncing from a private bucket**, you must fill the **AWS Access Key ID** and **AWS Secret Access Key** fields with the appropriate credentials to authenticate the connection. All other fields are optional and can be left empty. Refer to the [S3 Provider Settings section](#s3-provider-settings) below for more information on each field.
```

Comment:
could you make a PR to the S3 bucket adding the [`always_show: true` annotation ](https://docs.airbyte.com/connector-development/connector-specification-reference/#collapsing-optional-fields)? These fields are pretty common so I think they shouldn't be hidden under the optional fields section

Reply:
Will do!